### PR TITLE
Add feature template.

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Add feature template.

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: .github/FEATURE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+**Description of Problem**:
+Short overview of the current situation. 
+Why is this feature needed?
+
+**Overview of the Solution**:
+How does a possible solution look like?
```
Comment (amn41) [inline .github/FEATURE_TEMPLATE.md]:
```suggestion
What would a possible solution look like?
```

File: .github/FEATURE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+**Description of Problem**:
+Short overview of the current situation. 
+Why is this feature needed?
```
Comment (amn41) [inline .github/FEATURE_TEMPLATE.md]: maybe suggest to link to any relevant forum threads

Comment (tabergma) [reply .github/FEATURE_TEMPLATE.md]: I would put them under the section `example`. We could add the suggestion over there. Maybe we should rename that section to `resources` or something like that to make it more clear.

**Comment by tabergma** [comment]
part of https://github.com/RasaHQ/rasa/pull/3705

**Pull request closed without merging**